### ISSUE TITLE
Bug: windows defender warning is not fixed by recommended script

### ISSUE THREAD

**Issue by dev-nicolaos** [lukesampson/scoop#3673]
# Environment
- Windows 10 version 1903 build 18362.387
- Scoop latest commit = 3e55a709
- Example commands run in powershell being run as administrator

# Steps To Reproduce
1. `scoop checkup`
2. observe warning "WARN  Windows Defender may slow down or disrupt installs with realtime scanning."
3. Observe recommendation "Consider running:
    sudo Add-MpPreference -ExclusionPath 'C:\Users\username\scoop'
  (Requires 'sudo' command. Run 'scoop install sudo' if you don't have it.)"
4. Run `sudo Add-MpPreference -ExclusionPath 'C:\Users\username\scoop'`
5. Observe script exit with no error.
6. Run `scoop checkup` again.

# Expected behavior
Warning about Windows Defender is gone having run recomended script to address issue.

# Actual behavior
Warning is still present

# Additional notes
I actually get two of these warnings when running `scoop checkup` the one listed above and an identical one for "C:\ProgramData\scoop". Both behave the same way. I've also tried restarting my PC after running the fix scripts but it seems to have no effect.

**Comment by h404bi** [comment]
Have you tried to open an admin session of PowerShell and ran `Add-MpPreference -ExclusionPath 'C:\Users\<yourname>\scoop'`?

**Comment by dev-nicolaos** [comment]
> 
> 
> Have you tried to open an admin session of PowerShell and ran `Add-MpPreference -ExclusionPath 'C:\Users\<yourname>\scoop'`?

Yes, as I said, all commands were run in an admin session of Powershell. I tried first without sudo, then installed it and ran it with sudo. Same result either way.

**Comment by trip23app** [comment]
update **steps to reproduce** to make it clear that you ran the command in step 4 twice. once for each folder.

<hr>

I did `Add-MpPreference -ExclusionPath F:\scoop` ; it worked when I opened powershell 5.1  session  as admin. 

I checked it using the Windows Security app. 
steps:- Open the app and then Virus and threat protections, then 'manage settings'  then 'add or remove exclusions'.

**Comment by trip23app** [comment]
did you try double quotes like `Add-MpPreference -ExclusionPath "C:\Temp"`

and to check if it worked using powershell, try 
```
$Preferences = Get-MpPreference
$Preferences.ScanScheduleDay
```
before and after doing the `Add-MpPreference` part.

if you make a mistake, undo it using https://docs.microsoft.com/en-us/powershell/module/defender/remove-mppreference?view=win10-ps#examples

<hr>

Also, don't worry about this exact warning, you can still use scoop. It **maybe** slow, but I would not exclude anything if I'm going to be installing scoop or any new app for the first time. Also, there are no guarantees about the slowdown or it being perceivable.

**Comment by dev-nicolaos** [comment]
Figured it out. The recommended script has no effect because the computer I encountered the issue on is a company-owned machine, and exclusions are managed by the group policy. Apparently even though I have local admin rights, the group policy can prevent adding exclusions.

Maybe this could be documented in the warning so that others aren't confused over the same thing? That being said, I could see that getting pretty verbose, so I'd be OK just closing the issue with no change.
